### PR TITLE
[pt-br] Update tutorials/kubernetes-basics/create-cluster

### PR DESCRIPTION
**Pull request by stormqueen1990**
### Changes
* Update the cluster-intro tutorial to reflect the latest English version and remove Katacoda references.
* Update the create-cluster/_index.md page to reflect the latest English version.

### CODE REVIEW

**Comment by MrErlison** [comment]
Hi @stormqueen1990!

Thanks again for you great job!

/lgmt

**Comment by MrErlison** [comment]
/cc @jcjesus 

Could you please provide a second review?

**Comment by jcjesus** [comment]
> /cc @jcjesus
> 
> Could you please provide a second review?

Sure!

**Review by jcjesus — CHANGES_REQUESTED** [review]
Hi @MrErlison, good job, there is just only typo to fix.
Thanks

File: content/pt-br/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html [hunk 1]
```diff
@@ -75,33 +103,85 @@ <h2 style="color: #3771e3;">Diagrama de Cluster</h2>
...
         <div class="row">
             <div class="col-md-8">
-                <p>Ao implantar aplicativos no Kubernetes, você diz à camada de gerenciamento para iniciar os contêineres de aplicativos. A camada de gerenciamento agenda os contêineres para serem executados nos nós do cluster. <b> Os nós se comunicam com o camada de gerenciamento usando a <a href="/docs/concepts/overview/kubernetes-api/"> API do Kubernetes </a> </b>, que a camada de gerenciamento expõe. Os usuários finais também podem usar a API do Kubernetes diretamente para interagir com o cluster.</p>
+                <p>
+                  Ao implantar aplicativos no Kubernetes, você solicita que a camada
+                  de gerenciamento inicie os contêineres de aplicativos. A camada
+                  de gerenciamento aloca os contêineres para serem executados nos
+                  nós do cluster.
+                  <b>
+                    Os nós se comunicam com o camada de gerenciamento usando a
```
[the remark this answers is not in the document]
Comment (stormqueen1990) [reply content/pt-br/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.html]: Hi there, @jcjesus! This PR was actually opened by me 😅
Just fixed this!

**Pull request merged**